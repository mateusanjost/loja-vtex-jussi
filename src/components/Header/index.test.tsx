import TestRenderer from "react-test-renderer";
import HeaderComponent from "./index";

describe("Header", () => {
  it("should match snapshot", () => {
    const testRenderer = TestRenderer.create(<HeaderComponent />);
    const jsonComponent = testRenderer.toJSON();

    expect(jsonComponent).toMatchSnapshot();
  });
});
