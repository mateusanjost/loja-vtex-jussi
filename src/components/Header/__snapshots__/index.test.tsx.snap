// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<header
  className="sc-bdnxRM kPkVeH"
>
  <div
    className="sc-gtsrHT iKUmUO"
  >
    <img
      alt="Logo Jüssi - The Business Agency"
      src="logoJussiVectorGreen.svg"
    />
  </div>
  <nav
    className="sc-dlnjwi dJJCXy"
  >
     
    <div
      className="sc-eCApnc howTjG"
    >
      <a
        href="https://jussi.com.br/services.html"
        rel="noreferrer noopener"
        target="_blank"
      >
        Nossas soluções
      </a>
      <a
        href="https://jussi.com.br/"
        rel="noreferrer noopener"
        target="_blank"
      >
        Conheça a Jüssi
      </a>
    </div>
    <div
      className="sc-jSFjdj cnusMS"
    >
      <input
        className="sc-hKFxyN aigui"
        placeholder="Buscar"
        type="search"
      />
      <div
        className="sc-gKAaRy djnqAS"
      >
        <a
          href="/"
          target="_blank"
        >
          Login
        </a>
        <img
          alt="Shopping Cart - Jussi"
          src="shopping-cart.svg"
        />
      </div>
    </div>
  </nav>
</header>
`;
